### PR TITLE
Improve pyrex cython linter.

### PR DESCRIPTION
**Pull request by nicopauss**
Like many other linters, use variables for the executable and options
used by the linter.
By default, the linter now report every warnings as errors with
`--warning-errors`.
Also add include directory and set working directory to file directory.

### CODE REVIEW

File: ale_linters/pyrex/cython.vim [hunk 1]
```diff
@@ -1,10 +1,27 @@
-" Author: w0rp <devw0rp@gmail.com>
+" Author: w0rp <devw0rp@gmail.com>,
+" Nicolas Pauss <https://github.com/nicopauss>
 " Description: cython syntax checking for cython files.
 
+call ale#Set('pyrex_cython_executable', 'cython')
+call ale#Set('pyrex_cython_options', '--warning-extra --warning-errors')
```
Comment (w0rp) [inline ale_linters/pyrex/cython.vim]: Could you remove `--warning-errors` here, so we can preserve the current behavior? If users want warnings to be errors, they can change the option themselves. I don't want to change the behavior without a very compelling reason, and whether warnings should be errors is subjective.

Comment (nicopauss) [reply ale_linters/pyrex/cython.vim]: Done.

**Comment by nicopauss** [comment]
Done

**Comment by w0rp** [comment]
Good stuff. :+1: Cheers! :beers:

**Pull request merged**